src/store/PrivateTopicStore.ts

```diff
@@ -15,7 +15,7 @@ export default class NetworkStore implements Store {
     const contents = (
       await this.waku.store.queryHistory([buildUserPrivateStoreTopic(key)], {
         pageSize: 1,
-        pageDirection: PageDirection.FORWARD,
+        pageDirection: PageDirection.BACKWARD,
         callback: function (msgs: WakuMessage[]): boolean {
           return Boolean(msgs[0].payload)
```

Comment:
I guess we don't need to rush into a fix for malicious/broken bundles written to the private store topic. But we should probably figure out what our strategy is there.